dashboard/app/views/levels/_admin.html.haml

```diff
@@ -73,6 +73,19 @@
             = link_to build_script_level_path(script_level), build_script_level_path(script_level)
       %br/
       .project_admin
+      %br/
+      - if params[:channel_id]
+        - _, channel_id = storage_decrypt_channel_id(params[:channel_id])
+        - featured = FeaturedProject.exists?(:storage_app_id => channel_id)
+        - featured_project_id = featured ? FeaturedProject.find_by_storage_app_id(channel_id).id : nil
+        #unfeature_project{'style' => ('display: none;' unless featured)}
+          %button{id: "feature", class: "btn btn-default btn-sm"}
+            Stop featuring in gallery
+        #feature_project{'style' => ('display: none;' if featured)}
+          %button{id: "feature", class: "btn btn-default btn-sm"}
+            Feature in gallery
+        = content_tag 'div', '', class: 'featured_project_id', data: {id: featured_project_id}
```

Comment:
per other comments, you shouldn't need to obtain and set featured_project_id here